File: Test/m/matrix_operation.m
function [horiz_cat, vert_cat, sorted_cols, sorted_rows, sorted_rows_array, I, unsqueezed, row, corner] = matrix_operations(a, b, M)
    % horiz_cat = [a b];
    % vert_cat = [a; b];
    sorted_cols = sort(a);
    sorted_rows = sort(a, 2);
    [sorted_rows_array, I] = sortrows(a, 1);
    unsqueezed = squeeze(a);
    row = M(2, :);
    corner = M(end, end);
end

File: Test/py/matrix_operation.py
from numpy import *

def matrix_operations(a, b, M):
    # horiz_cat = concatenate((a, b), 1)
    # vert_cat = concatenate((a, b))
    sorted_cols = sort(a)
    sorted_rows = sort(a, axis=1)
    I = argsort(a[:, 0])
    sorted_rows_array = a[I, :]
    unsqueezed = a.squeeze()
    row = M[1, :]
    corner = M[-1, -1]
    return horiz_cat, vert_cat, sorted_cols, sorted_rows, sorted_rows_array, I, unsqueezed, row, corner